.github/workflows/build-and-deploy-java.yml

```diff
@@ -54,10 +54,15 @@ jobs:
         run: |
           # Print Info
           java -version
-          gradle --version
+          GRADLE_VERSION=$(gradle --version | awk '/^Gradle/ {print $2}' | head -1)
```

Comment:
Instead of using hard-coded Gradle versions, we extract the version number from the current release.